src/index.js

```diff
@@ -327,24 +327,27 @@ window["StatsigSidecar"] = window["StatsigSidecar"] || {
 
     try {
       const user = window?.statsigUser ?? (
-        overrideUserID ? { 
+        overrideUserID ? {
             userID: overrideUserID,
             customIDs: { stableID: overrideUserID }
-          } 
+          }
           : {}
       );
       const options = window?.statsigOptions ?? {};
       options.disableLogging = !autoStart;
       if (initUrlOverride || logUrlOverride) {
-        options.networkConfig = { 
+        options.networkConfig = {
           initializeUrl: initUrlOverride,
           logEventUrl: logUrlOverride,
         };
       }
       this._statsigInstance  = new StatsigClient(apiKey, user, options);
       await this._statsigInstance.initializeAsync();
-      runStatsigAutoCapture(this._statsigInstance);
-      
+
+      if (window?.runStatsigAutoCapture !== false) {
```

Comment:
This is notably different from [this check](https://github.com/statsig-io/sidecar-js/pull/5/files#diff-bfe9874d239014961b1ae4e89875a6155667db834a410aaaa2ebe3cf89820556R349). That's because Auto-Capture is currently enabled by default, so I didn't want to disable unless explicitly told. As apposed to the Session-Replay, which isn't currently enabled by default, so I require it to be explicitly turned on.

I.e. opt-out vs opt-in